src/Maestro/tests/Maestro.ScenarioTests/Maestro.ScenarioTests.csproj

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <Nullable>enable</Nullable>
```

Comment:
><Nullable>enable</Nullable> [](start = 4, length = 27)

I don't like 1 library having this enabled. We should do it everywhere or nowhere, otherwise it's confusing to refactor code (I vote for nowhere).

We need to stop making special snowflake projects, and only have a single way to build and deploy code.

Reply:
Done. Filed https://github.com/dotnet/arcade/issues/4706